crypto/s2n_drbg.c

```diff
@@ -129,8 +131,12 @@ int s2n_drbg_instantiate(struct s2n_drbg *drbg, struct s2n_blob *personalization
     memcpy_check(ps.data, personalization_string->data, MIN(ps.size, personalization_string->size));
 
     /* Seed / update the DRBG */
-    GUARD(s2n_drbg_seed(drbg));
+    GUARD(s2n_drbg_seed(drbg, &ps));
 
+    /* After initial seeding, pivot to RDRAND if available and not overridden */
+    if (drbg->entropy_generator == NULL && s2n_cpu_supports_rdrand()) {
+        drbg->entropy_generator = s2n_get_rdrand_data;
+    }
```

Comment:
Even with no RDRAND we will still have PR. Check out lines 94-99. If the entropy generator is not set in the s2n_drbg object, then it falls back to urandom.

https://github.com/awslabs/s2n/pull/248/files#diff-c03ac19053399e59511956db1159e2e8R98